src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -3609,6 +3610,8 @@ namespace HVACUnitarySystem {
 					errFlag = false;
 				}
 
+				UnitarySystem( UnitarySysNum ).HeatingCoilAvailSchPtr = GetCoilAvailScheduleIndex( HeatingCoilType, HeatingCoilName, errFlag );
+
```

Comment:
MultiStage heating coil never got the coil avail schedule. So it couldn't even turn on for SP based control.